src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -178,6 +178,17 @@ internal override IOperation GetOperationWorker(CSharpSyntaxNode node, Cancellat
                 case AccessorDeclarationSyntax accessor:
                     model = (accessor.Body != null || accessor.ExpressionBody != null) ? GetOrAddModel(node) : null;
                     break;
+
+                case CompilationUnitSyntax unit:
+                    // Allow getting IOperation tree for the entire simple program body by requesting a tree for any of the comilation units with top level statements.
```

Comment:
nit: comilation -> compilation

Reply:
> nit: comilation -> compilation

Already caught by Chuck.

---
In reply to: [384798117](https://github.com/dotnet/roslyn/pull/41926#discussion_r384798117) [](ancestors = 384798117)